src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,26 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.1-m1
+
+* `master` branches across all the projects are renamed to `devel`
+* Gateway engine is renamed to `submariner-gateway` from `submariner`
+* Helm charts support submariner deployments using the operator
+* Broker creation is managed by the operator inside of subctl
+* Operator and individual applications have separate service accounts with appropriate privileges
+* For Openshift, Lighthouse CoreDNS metrics are integrated with other CoreDNS metrics
+* KIND deployments support OVS CNI
+* `submariner_connections` is renamed to `submariner_requested_connections` in Prometheus for OpenShift clusters
+* Clusters can be connected with only Lighthouse and no dataplane connectivity via subctl
```

Comment:
```suggestion
* Clusters can be connected with only Lighthouse and no dataplane connectivity via `subctl deploy-broker --components`
```

Reply:
For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can deploy Service Discovery only via the `--components` flag

Reply:
IMHO the sentence is a bit confusing as we are saying Clusters can be "connected" with only Lighthouse.

How about something like...

"Support for deploying just Lighthouse for Service Discovery via subctl when dataplane is already connected by the respective cloud provider"